skillmap/package-lock.json

```diff
@@ -18536,9 +18547,11 @@
       }
     },
     "node_modules/type-fest": {
-      "version": "0.21.3",
-      "resolved": "https://registry.npmjs.org/type-fest/-/type-fest-0.21.3.tgz",
-      "integrity": "sha512-t0rzBq87m3fVcduHDUFhKmyyX+9eo6WQjZvf51Ea/M0Q7+T374Jp1aUiyUl0GKxp8M/OETVHSDvmkyPgvX+X2w==",
+      "version": "0.13.1",
```

Comment:
Why are we downgrading this package?

Reply:
I still see this downgrade

Reply:
oof, i remember making the changes yesterday, but am not sure what happened, thanks for catching this!